galaxy/wrapper/computeMatrix.xml

```diff
@@ -85,12 +79,14 @@
                 help="Label to use in the output."/>
         </repeat>
 
-        <param name="scoreFile" format="bigwig" type="data"
```

Comment:
Do you mind this change? Otherwise I can implement it. You don't want a repeat block if you insert >5 files also this enables a collection as input.